packages/shared/src/useApi/useApi.js

```diff
@@ -0,0 +1,28 @@
+// @flow strict
```

Comment:
Could you add a test or some README that would give an example on how to use it, and why you'd want to use it?
I know the article is great, but a quick example on how to use it could be beneficial.